install/kubernetes/generate-sidecar-injector-yaml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ROOT=$(cd $(dirname $0)/../../; pwd)
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+IN_TEMPLATE=${ROOT}/install/kubernetes/istio-sidecar-injector.yaml.template
+OUT_YAML=${ROOT}/install/kubernetes/istio-sidecar-injector.yaml
+
+CA_BUNDLE=$(kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 -w0)
```

Comment:
@sbezverk @ZackButcher - any thoughts on how this `client-ca-file` extraction step and the certificate creation/approval below might be integrated with helm? To summarize, we need to do the following per-installation after install files are downloaded from istio.io:

1) Generate a key with CN set to the k8s webhook service name. 
2) Sign the key generated in step #1.
3) Update the webhook configmap with the caBundle used to sign the key in step #2.

I used cfssl for #1 and CertificateSigningRequest API (k8s CA) for #2. For #3 I need to extract the caBundle from an existing k8s configmap.